src/frontend/Komponenter/Behandling/VedtakOgBeregning/Skolepenger/OpphørSkolepenger/OpphørUtgiftsperiodeSkolepenger.tsx

```diff
@@ -0,0 +1,127 @@
+import styled from 'styled-components';
+import React from 'react';
+import { SkolepengerUtgift } from '../../../../../App/typer/vedtak';
+import { Element } from 'nav-frontend-typografi';
+import FjernKnapp from '../../../../../Felles/Knapper/FjernKnapp';
+import { SkolepengerOpphørProps } from '../typer';
+import navFarger from 'nav-frontend-core';
+import {
+    formaterIsoMånedÅrFull,
+    formaterTallMedTusenSkilleEllerStrek,
+} from '../../../../../App/utils/formatter';
+import KansellerKnapp from '../../../../../Felles/Knapper/KansellerKnapp';
+import { locateIndexToRestorePreviousItemInCurrentItems } from '../utils';
+
+const Utgiftsrad = styled.div<{
+    lesevisning?: boolean;
+    erHeader?: boolean;
+    erFjernet?: boolean;
+}>`
+    display: grid;
+    grid-template-areas: 'fraOgMedVelger utgifter stønad';
+    grid-template-columns: ${(props) =>
+        props.lesevisning ? '9rem 4rem 4rem' : '12rem 5rem 5rem 4rem'};
+    grid-gap: 0.5rem;
+    margin-bottom: ${(props) => (props.erHeader ? '0rem' : '0.5rem')};
+    text-decoration: ${(props) => (props.erFjernet ? 'line-through' : 'inherit')};
+`;
+
+const FlexRow = styled.div`
+    display: flex;
+    margin-top: 0.75rem;
+`;
+
+const FlexColumn = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+const FargetStrek = styled.span`
+    border-left: 3px solid ${navFarger.navGra80};
```

Comment:
```suggestion
    border-left: 3px solid --var(--navds-global-color-gray-800);
```
Samme her med [nytt designsystem](https://aksel.nav.no/designsystem/side/farge-tokens/globale) (gray-800 er ikke nøyaktig den samme fargen som Gra80, da fargepaletten er endret til å inneholde flere nyanser).